x-pack/plugins/security_solution_serverless/public/get_started/welcome_panel/spacer.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { css } from '@emotion/react';
+import { useEuiTheme } from '@elastic/eui';
+
+/* <div> cannot appear as a descendant of <p>, EuiSpacer is a div */
+const SpacerComponent = ({
```

Comment:
Probably we can avoid duplicating EuiSpacer component only for this page. I see it is only used in 2 places, it would be better to solve the problem using CSS.